product_manufacturer_add_supplierinfo/__openerp__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
```

Comment:
would you kindly add the 2015-17 to the date? Thank you so much :medal_military: